### ISSUE TITLE
Dependency issue

### ISSUE THREAD

**Issue by rosnelclaire** [ankushaggarwal/pymecht#22]
There seems to be a dependency issue with torch. Please see attached screenshot. 
![Screenshot 2023-12-01 at 14 47 47](https://github.com/ankushaggarwal/pymecht/assets/95284801/7301aff4-ccea-4319-9384-d0ab8abc5067)

**Comment by ankushaggarwal** [comment]
Yes, I get the same notification. We could remove torch as a dependency, it is only used for quasi-random sampling.
